kdestandardize.ado

```diff
@@ -1414,49 +1423,247 @@ prog def kdestandardize
+	} // End of handling of the FEMALE_CNT variable
+	
+	//Handles instances of the TOTAL_STDNT_CNT variable
+	if `: list posof "total_stdnt_cnt" in x' != 0 {
+	qui: rename total_stdnt_cnt totstdcnt
+	qui: destring totstdcnt, replace ignore("*,-R %")
+	la var totstdcnt "Total Student Involvemnt Count"
+	} // End of handling of the TOTAL_STDNT_CNT variable
+	
+	//Handles instances of the TOTAL_UNIQUE_EVENT_CNT variable
+	if `: list posof "total_unique_event_cnt" in x' != 0 {
+	qui: rename total_unique_event_cnt totunqevntcnt
+	qui: destring totunqevntcnt, replace ignore("*,-R %")
+	la var totunqevntcnt "Total Unique Event Count"
+	} // End of handling of the TOTAL_UNIQUE_EVENT_CNT variable
+	
+	//Handles instances of the RPT_HEADER variable
+	if `: list posof "rpt_header" in x' != 0 {
+	qui: replace rpt_header=cond(`rx'(rpt_header, "Behavior Events", 1), "1", ///
+							  cond(rpt_header=="Behavior Events by Context", "2",        ///
+							  cond(rpt_header=="Behavior Events by Grade Level", "3",    ///
+							  cond(rpt_header=="Behavior Events by Locations"," 4",      ///
+							  cond(rpt_header=="Behavior Events by Socio-Economic Status", "5",  ///
+							  cond(rpt_header=="Discipline Resolutions", "6",            ///
+							  cond(rpt_header=="Discipline-Resolutions", "7",            ///
+							  cond(rpt_header=="Legal Sanctions", "8", ""))))))))        ///
+							  
+	qui: rename rpt_header rpthdr
+	qui: desting rpthdr,replace ignore("*,-R %")
+	la val rpthdr rpthdr
+	la var rpthdr Report Header
+	} // End of handling of the RPT_HEADER variable
+	
+	//Handles instances of the RPT_HEADER_ORDER variable
+	if `: list posof "rpt_header_order" in x' !=0 {
+	qui: rename rpt_header_order rpthdrodr
+	qui: destring rpthdrodr, replace ignore("*,-R %")
+	la var rpthdrodr "Report Header Order"
+	} // End of handling of the RPT_HEADER_ORDER variable
+	
+	//Handles instance of the RPT_LINE_ORDER variable
+	if `: list posof "rpt_line_order" in x' !=0 {
+	qui: rename rpt_line_order rptlnodr
+	qui: destring rptlnodr, replace ignore("*,-R %")
+	la var rptlnodr "Report Line Order"
+	} // End of handling of the RPT_LINE_ORDER variable
+	
+	//Handles instance of the SPENDING_PER_STDNT variable
+	if `: list posof "spending_per_stdnt" in x' !=0 {
+	qui: rename spending_per_stdnt spndprstdnt
+	qui: destring spndprstdnt, replace ignore("*,-R %")
+	la var spndprstdnt "Spending Per Student"
+	} // End of handling of the SPENDING_PER_STDNT variable
+	
+	//Handles instance of the AVG_DAILY_ATTENDANCE variable
+	if `: list posof "avg_daily_attendance" in x' !=o {
+	qui: rename avg_daily_attendance avgdlyatnd
+	qui: destring avgdlyatnd, replace ignore("*,-R %")
+	la var avgdlyatnd "Average Daily Attendance"
+	} // End of handling of the AVG_DAILY_ATTENDANCE variable
+	
+	//Handles instance of the MEMBERSHIP_TOTAL variable
+	if `: list posof "membership_total" in x' !=0 {
+	qui: rename membership_total membrshptot
```

Comment:
Use `membership` for this variable and `"Total Number of Students"` or something equivalent (and the same as what is used previously in the program) for the variable label.